### ISSUE TITLE
Entropy loss bug in Lookup Free Quantization

### ISSUE THREAD

**Issue by flm8620** [myscience/open-genie#9]
https://github.com/myscience/open-genie/blob/732b9f9b746f18fff1a0fb22f83638224f2f7cc6/genie/module/quantization.py#L125

```python
entropy_loss = inp_ent + self.diversity_weight * avg_ent
```

This line should be 

```python
entropy_loss = inp_ent - self.diversity_weight * avg_ent
```
